src/features/patients/pages/PatientForm.tsx

```diff
@@ -1,44 +1,55 @@
-import { FC, useState } from 'react';
-import { Backdrop, Button, CircularProgress, Paper } from '@material-ui/core';
-import { createStyles, makeStyles } from '@material-ui/core/styles';
-import { ConfirmationDialog } from '../../../shared/dialog';
-import { JsonSchemaForm } from '../../../shared/components';
-import patientSchema from '../../../json/patient-schema.json';
-import patientDetailsSchema from '../../../json/patient-details-schema.json';
-import patientUI from '../../../json/patient-ui.json';
-import patientDetailsUI from '../../../json/patient-details-ui.json';
-import { usePatientSchema } from '../hooks/usePatientSchema';
+import { FC, useState } from "react";
+import {
+  Backdrop,
+  Box,
+  Button,
+  CircularProgress,
+  Paper,
+} from "@material-ui/core";
+import { createStyles, makeStyles } from "@material-ui/core/styles";
+import { ConfirmationDialog } from "../../../shared/dialog";
+import { JsonSchemaForm } from "../../../shared/components";
+import { usePatientSurveySchemaQuery } from "../hooks/usePatientSurveySchemaQuery";
+import { usePatientSchemaQuery } from "../hooks/usePatientSchemaQuery";
 
 const useStyles = makeStyles(() =>
   createStyles({
     backdrop: {
       zIndex: 100,
-      color: '#fff',
+      color: "#fff",
```

Comment:
time for `.prettierrc` maybe!

Reply:
Ha. yeah. Right after we get the full production ready app goin, right? 🤣